docs/pages/repo/docs/reference/command-line-reference/optimize.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "turbo optimize"
+description: Enable the fastest defaults for your Turborepo with one, interactive command.
+
+---
+
+# `turbo optimize`
+
+Enable the fastest defaults for your Turborepo with one, interactive command.
+
+// image
+
+You'll be taken through a short series of steps to enable the fastest settings for your Turborepo while you're working locally. These optimizations include:
+
+- **Git FS Monitor**: `turbo` leans on Git to do file change discovery. Since we have to wait for `git` to tell us about changes, we can use [Git's built-in filesystem monitor](https://git-scm.com/docs/git-fsmonitor--daemon) to get those notifications sooner.
+- **Turbo Daemon**: Turborepo has its own daemon to do work for optimistically understanding your repository in the background. By doing the shared, common work that `turbo` will need to do when you run your tasks beforehand, we can make `turbo` much faster.
+- **Remote Caching**: While Turborepo can cache your work locally, it can also share a cache across all of your machines. Enabling [Remote Caching](/repo/docs/core-concepts/remote-caching) makes your caching **multiplayer**, ensuring that you, your teammates, and your CI machines, never do the same work twice.
+- **Check `turbo` version**: We're always working towards making `turbo` better. To ensure you are using the latest version of Turborepo, we'll check your version and remind you to install `latest` if you aren't using it yet.
```

Comment:
This looks great. Last diag I added is for checking if the LSP / extension is installed.

```suggestion
- **Check `turbo` version**: We're always working towards making `turbo` better. To ensure you are using the latest version of Turborepo, we'll check your version and remind you to install `latest` if you aren't using it yet.
- **Check for `turbo` extension**: The extension will ensure the daemon is running as soon as you launch your project as well as providing syntax highlighting and suggestions in your `turbo.json`.
```